save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -2,29 +2,30 @@ package org.cqfn.save.backend.service
 
 import org.cqfn.save.backend.repository.TestExecutionRepository
 import org.cqfn.save.entities.TestExecution
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 
 /**
  * Service for test result
  */
 @Service
 class TestExecutionService(private val testResultRepository: TestExecutionRepository) {
+    private val log = LoggerFactory.getLogger(TestExecutionService::class.java)
+
     /**
      * @param testExecutions list of test executions
      */
     fun saveTestResult(testExecutions: List<TestExecution>) {
         testExecutions.forEach { testExec ->
-            testResultRepository.run {
-                val foundTestExec = this.findById(testExec.id)
-                 if (!foundTestExec.isPresent) {
-                     println("error")
-                 } else {
-                     foundTestExec.map {
-                         it.startTime = testExec.startTime
-                         it.endTime = testExec.endTime
-                         it.status = testExec.status
-                         this.save(it)
-                     }
+            val foundTestExec = testResultRepository.findById(testExec.id)
+            if (!foundTestExec.isPresent) {
```

Comment:
Oh, it's `Optional` from Java, I forgot it. Then it has useful methods like `ifPresentOrElse`